### PR TITLE
Update passing-data.mdx

### PR DESCRIPTION
**Pull request by githubeing**

### CODE REVIEW

**Comment by timneutkens** [comment]
Doesn't seem like this is needed 🤔

**Comment by githubeing** [comment]
> Doesn't seem like this is needed

"we import the function this will inject the router" - isn't a comma or something missing here? there are two subjects here: "we" and "this" and two predicates - "import" and "will inject". don't they need a comma or an "and" word or smth?

File: learn/pages/learn/basics/create-dynamic-pages/passing-data.mdx [hunk 1]
```diff
@@ -61,7 +61,7 @@ Now our page will look like this:
 
 Here's what's happening in the above code.
 
-- We import and use the "withRouter" function from "next/router" this will inject the Next.js router as a property
+- We import and use the "withRouter" function from "next/router" -- this will inject the Next.js router as a property
```
Comment (timneutkens) [inline learn/pages/learn/basics/create-dynamic-pages/passing-data.mdx]:
```suggestion
- We import and use the "withRouter" function from "next/router" which will inject the Next.js router as a property
```

**Comment by timneutkens** [comment]
Yeah the wording can be improved! Added a suggestion 🙏

File: learn/pages/learn/basics/create-dynamic-pages/passing-data.mdx (hunk 1 above)
Comment (githubeing) [reply learn/pages/learn/basics/create-dynamic-pages/passing-data.mdx]: this variant is also possible. btw, the period is missing at the end of the sentence.

**Comment by timneutkens** [comment]
Thanks!

**Pull request merged**